src/Surfnet/StepupSelfService/SelfServiceBundle/Service/U2fSecondFactor/ProofOfPossessionResult.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+/**
+ * Copyright 2015 SURFnet bv
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupSelfService\SelfServiceBundle\Service\U2fSecondFactor;
+
+use Surfnet\StepupSelfService\SelfServiceBundle\Exception\LogicException;
+use Surfnet\StepupU2fBundle\Dto\Registration;
+use Surfnet\StepupU2fBundle\Service\RegistrationVerificationResult;
+
+/**
+ * @SuppressWarnings(PHPMD.TooManyMethods)
+ */
+final class ProofOfPossessionResult
+{
+    /**
+     * @var string|null
+     */
+    private $secondFactorId;
+
+    /**
+     * @var \Surfnet\StepupU2fBundle\Service\RegistrationVerificationResult|null
+     */
+    private $wrappedResult;
```

Comment:
something something duck something - aka please just name it the `registrationVerificationResult`